### ISSUE TITLE
Difference in output layer shape between pytorch model and js model

### ISSUE THREAD

**Issue by drorpa** [castorini/howl-models#3]
Hi,

I'm trying to use howl and I have a question regarding the pretrained models.

When comparing the weights for the js model at _howl-deploy/JavaScript/hey_firefox.js_
and the weights for the pytorch model at _howl/hey-fire-fox/model.pt.bin_ (or _howl/hey-fire-fox/model-best.pt.bin_),
I noticed that for the js model, output.weight has shape of [10, 45] and output.bias has shape of [10],
while for the pytorch model, output.weight has shape of [4, 45] and output.bias has shape of [4].

From my understanding of the model, the output layer should have 4 neurons ['hey', 'fire', 'fox', OOV],
so my question is why is there a difference in the shapes of the output layers?

I assumed that the weights in hey_firefox.js are from the model trained in pytorch,
if so, can you share details on the process?
and if not, can you share where the weights in hey_firefox.js are from?

**Comment by daemon** [comment]
As far as I remember, the last 6 of those labels are unused. @ljj7975 do you know?
